src/DynamoPackagesUI/Views/PackageManagerView.xaml.cs

```diff
@@ -39,23 +39,23 @@ public PackageManagerView(PackageManagerViewModel viewModel)
 
             InitializeComponent();
 
-            viewModel.PkgMgrCommands.ParentWindow = this;
+            viewModel.ParentWindow = this;
             //cefHelper object for Explore Packages, Explore Authors and My Packages Tab
-            this.cefBrowser.RegisterJsObject("cefHelper", viewModel.PkgMgrCommands);
+            this.cefBrowser.RegisterJsObject("cefHelper", viewModel);
```

Comment:
we should consider renaming `cefHelper` to `packageUtils` or something else as commented in another review.